zetaclient/zetasupplychecker/zeta_supply_checker.go

```diff
@@ -1,8 +1,14 @@
-package zetaclient
+package zetasupplychecker
```

Comment:
I think we can name the package `supplychecker` in case we add new functionalities in the future